packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -886,42 +812,21 @@ contract('FederatedAttestations', (accounts: string[]) => {
     it('should succeed when >1 attestations are registered for (identifier, issuer)', async () => {
       const account2 = accounts[3]
       await signAndRegisterAttestation(identifier1, issuer1, account2, nowUnixTime, signer1)
-      const deleteAttestation = await federatedAttestations.deleteAttestation(
-        identifier1,
-        issuer1,
-        account2,
-        {
-          from: account2,
-        }
-      )
-      assertLogMatches2(deleteAttestation.logs[0], {
-        event: 'AttestationDeleted',
-        args: {
-          identifier: identifier1,
-          issuer: issuer1,
-          account: account2,
-        },
+      await checkStorageState(identifier1, issuer1, 1, account2, nowUnixTime, signer1, 0)
+      await federatedAttestations.deleteAttestation(identifier1, issuer1, account2, {
+        from: account2,
       })
+      await checkStorageState(identifier1, issuer1, 0, account1, nowUnixTime, signer1, 0)
     })
 
     it('should succeed when >1 identifiers are registered for (account, issuer)', async () => {
       await signAndRegisterAttestation(identifier2, issuer1, account1, nowUnixTime, signer1)
-      const deleteAttestation = await federatedAttestations.deleteAttestation(
-        identifier2,
-        issuer1,
-        account1,
-        {
-          from: account1,
-        }
-      )
-      assertLogMatches2(deleteAttestation.logs[0], {
-        event: 'AttestationDeleted',
-        args: {
-          identifier: identifier2,
-          issuer: issuer1,
-          account: account1,
-        },
+      await checkStorageState(identifier2, issuer1, 0, account1, nowUnixTime, signer1, 1)
+      await federatedAttestations.deleteAttestation(identifier2, issuer1, account1, {
+        from: account1,
       })
+      checkAttestationDeleted(identifier2, issuer1, account1, 0, 1)
```

Comment:
I like that in the previous case you check the storage state of the other attestation to make sure it's untouched. Can you do this here as well + in the other following tests? i.e. both:
- checkAttestationDeleted (for attestation that was deleted)
- checkStorageState (for attestation that was already there, to make sure it wasn't touched)